module-2/backend/week_two.md

```diff
@@ -4,29 +4,51 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 
 Note: When you're done, submit a PR. 
 
-1. At a high level, what is ActiveRecord? What does it do/allow you to do?
-2. What kind of methods are `belongs_to`, and `has_many`? (i.e. class or instance) Give an example.
-3. What do they allow you to do?
-4. What's the difference between agile workflow and waterfall method?
-5. What is the difference between `#new` and `#create`?
-6. At a basic level, what does cURL allow you to do?
-7. In a database that's holding students and teachers, what will be the relationship between students and teachers? Draw the schema diagram.
-8. Define foreign key, primary key, and schema.
-9. Describe the relationship between a foreign key on one table and a primary key on another table.
-10. What are the parts of an HTTP response?
-11. Describe some techniques to make our Sinatra code more DRY. Give an example of when you would use these techniques.
+1. At a high level, what is ActiveRecord? What does it do/allow you to do?  
+ActiveRecord is a Ruby gem that helps to simplify interactions with the database. It allows us to use keywords instead of building our own SQL query strings. Also helps us to set up and manage database table associations through short keywords.
+2. What kind of methods are `belongs_to`, and `has_many`? (i.e. class or instance) Give an example.  
+They are class methods that help us to set up one-to-many and many-to-many associations between database tables. Bike-share project related example:  
+one station could be associated with more than one trip => so within our `Station` model we had a `has_many: trips` association, which could be called as `station.trips` on any instance of of `station`.
```

Comment:
One note: I think you're getting this based on the capitalization in your answer, but you can only call `.trips` on an *instance* of station. So, you wouldn't be able to call `Station.trips`, you would need to first run `station = Station.find(1)`, and then you could run `station.trips`.